diff --git a/node_modules/next/dist/esm/server/lib/incremental-cache/fetch-cache.js b/node_modules/next/dist/esm/server/lib/incremental-cache/fetch-cache.js
index 3c4a9f6..b6d2274 100644
--- a/node_modules/next/dist/esm/server/lib/incremental-cache/fetch-cache.js
+++ b/node_modules/next/dist/esm/server/lib/incremental-cache/fetch-cache.js
@@ -107,6 +107,9 @@ export default class FetchCache {
                     },
                     next: fetchParams
                 });
+                console.log("[fetch-cache] get:", `${this.cacheEndpoint}/v1/suspense-cache/${key}`)
+                console.log("[fetch-cache] get: cached response", await res.text())
+                console.log("[fetch-cache] get: cached response headers", JSON.stringify(Object.fromEntries([...res.headers])))
                 if (res.status === 404) {
                     if (this.debug) {
                         console.log(`no fetch cache entry for ${key}, duration: ${Date.now() - start}ms`);
@@ -199,6 +202,14 @@ export default class FetchCache {
                     body: body,
                     next: fetchParams
                 });
+                console.log("[fetch-cache] set:", `${this.cacheEndpoint}/v1/suspense-cache/${key}`)
+                console.log("[fetch-cache] set: input headers", JSON.stringify({
+                    ...headers,
+                    "X-Vercel-Cache-Item-Name": fetchUrl || ""
+                }))
+                console.log("[fetch-cache] set: input next", JSON.stringify(fetchParams))
+                console.log("[fetch-cache] set: response", await res.text())
+                console.log("[fetch-cache] set: response headers", JSON.stringify(Object.fromEntries([...res.headers])))
                 if (!res.ok) {
                     this.debug && console.log(await res.text());
                     throw new Error(`invalid response ${res.status}`);
